pkg/storage/stores/metrics.go

```diff
@@ -6,16 +6,15 @@ import (
 )
 
 type metrics struct {
-	// ToDo(Sandeep): Refactor code to include index write requests
-	indexQueryLatency *prometheus.HistogramVec
+	storeRequestLatency *prometheus.HistogramVec
 }
 
 func newMetrics(reg prometheus.Registerer) *metrics {
 	return &metrics{
-		indexQueryLatency: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+		storeRequestLatency: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
 			Namespace: "loki",
-			Name:      "index_request_duration_seconds",
-			Help:      "Time (in seconds) spent in serving index query requests",
+			Name:      "store_request_duration_seconds",
```

Comment:
The changes from #6880 never made it into a release, correct?

Reply:
Is this metric only for index stores?

Reply:
> The changes from #6880 never made it into a release, correct?
yes, latest release `2.6.1` doesn't have it.

> Is this metric only for index stores?
All the read requests here are purely index queries, while the write request i.e `Put` and `PutOne`, uploads the chunk and indexes it as well. It would be hard to purely track just the index write request without refactoring the code.

Reply:
Yeah I think having a metric named `loki_store_request_duration_seconds` that doesn't include chunk store reads but only includes writes will be quite confusing.

How hard do you anticipate the refactoring would be? I want to try optimise for clarity as much as possible given that our system is already very complex.

Reply:
I looked at the code, and it seems quite complex and would take time. The other option would be to keep the old metric `loki_index_request_duration_seconds` which tracks just the index requests for now, and update just the reads dashboards until we refactor the code. I will keep the refactoring work on my to-do list until then.

Reply:
>  The other option would be to keep the old metric loki_index_request_duration_seconds which tracks just the index requests for now, and update just the reads dashboards until we refactor the code. I will keep the refactoring work on my to-do list until then.

Thanks Sandeep, I think that might work out better for our users - if you don't mind?